internal/provider/provider.go

```diff
@@ -152,7 +150,7 @@ func (p *MetalstackCloudProvider) Configure(ctx context.Context, req provider.Co
 		BaseURL:   apiUrl,
 		Token:     apiToken,
 		UserAgent: "terraform-provider-metal/" + p.version,
-		Debug:     viper.GetBool("debug"),
+		Debug:     true,
```

Comment:
@vknabel Do you have an idea, how to replace this?

Reply:
In `main.go` there is a local `debug` variable which you could make global.